internal/node/config.go

```diff
@@ -26,14 +28,16 @@ func LoadNodeConfiguration(configFilepath string) (*models.NodeConfiguration, er
 		config.WorkloadTypes = models.DefaultWorkloadTypes
 	}
 
-	// TODO-- audit for *string
+	if strings.EqualFold(runtime.GOOS, "windows") && !config.NoSandbox {
```

Comment:
see previous thing that makes me worried... runtime check for windows os rather than using build tags and/or xplat library